### PR TITLE
Key insights for Jails view

### LINKED ISSUE #40: Support key insights cards for new Jails view
**Is your feature request related to a problem? Please describe.**
As described in the new "Category 2: Jails" section of [the PRD](https://docs.google.com/document/d/1eOiMzk8_eDjC4W0xn89qlkmj_6SOqi1dBTyEusAGGaI/edit?ts=5fb45edc#heading=h.s7ludzwvb7fe), we are going to create a new Jails view alongside the existing Corrections view. The top section of this view, just below the new tab selector from #38, is a set of key insight cards about the jails system.

**Describe the solution you'd like**
The design of these cards is identical to the key insight cards from the existing corrections view. But the actual metrics shown are different, and one of these cards does _not_ have a percentage change component, so that will need to be supported.

Additionally, the captions below the cards are a bit more variable and less formulaic than they were in the corrections view.

**Describe alternatives you've considered**
N/A

**Additional context**
See the new "Category 2: Jails" section of [the PRD](https://docs.google.com/document/d/1eOiMzk8_eDjC4W0xn89qlkmj_6SOqi1dBTyEusAGGaI/edit?ts=5fb45edc#heading=h.s7ludzwvb7fe) and the Figma mocks it links to.

### PR DESCRIPTION
**Pull request by nasaownsky**
## Description of the change

> Support key insights cards for new Jails view

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #40

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/utils/toInt.js
Comment (Wmaileq) [inline src/utils/toInt.js:17]: If you look at the name of the functions expected output is an integer, not float.
Why do you need this change?

File: craco.config.js
Comment (Wmaileq) [inline craco.config.js:7]: I think we need to use JSON lines loader only for data in `data` folder to avoid possible pitfalls e.g. when we will need to import regular JSON.

File: src/components/MainPage/MainPage.js
Comment (Wmaileq) [inline src/components/MainPage/MainPage.js:43]: To make props consistent move `Data` word to the end of the name. `jailsKeyInsightsData` or `keyInsightsJailsData`. I think the better way is to start with DOMAIN (Jails, Corrections) then component `keyInsights`, `flowDiagram`.

File: src/components/shared/Card/Card.js
Comment (Wmaileq) [inline src/components/shared/Card/Card.js:76]: This is not the solution.
`Card` component is the UI component and should just render provided props.
In addition: it is a very unreliable check as the title could be changed later.

I think the better way is to move `formatNumber` logic to utils and render here just `{number}`.

File: src/components/shared/Card/Card.js
Comment (Wmaileq) [inline src/components/shared/Card/Card.js:78]: The same, let think of better solution.

File: src/utils/generateJailsKeyInsightsData.js
Comment (Wmaileq) [inline src/utils/generateJailsKeyInsightsData.js:42]: We can remove `() =>` here.

**Review by Wmaileq — CHANGES_REQUESTED** [review]

File: src/utils/toInt.js
Comment (nasaownsky) [reply src/utils/toInt.js:17]: Because in `value` of `getNormalizedStateData` could be floats (aka percentage numbers). I could create new `toFloat` function and use it on `value`.

File: src/components/MainPage/MainPage.js
Comment (nasaownsky) [reply src/components/MainPage/MainPage.js:43]: fixed

File: src/utils/generateJailsKeyInsightsData.js
Comment (nasaownsky) [reply src/utils/generateJailsKeyInsightsData.js:42]: fixed

File: src/components/shared/Card/Card.js
Comment (nasaownsky) [reply src/components/shared/Card/Card.js:76]: It is already in utils. I ended up with another solution using flag `isNumberPercent`.

File: src/components/shared/Card/Card.js
Comment (nasaownsky) [reply src/components/shared/Card/Card.js:78]: See comment above.

**Comment by jessex** [comment]
Welcome, @nasaownsky! Glad to have you on board.

File: craco.config.js
Comment (jessex) [inline craco.config.js:7]: Why did we switch to `.jsonl` files explicitly here? Did @colincadams approve that change previously and I just missed it?

File: src/App.js
Comment (jessex) [inline src/App.js:89]: In cases where we are creating a new parallel function for _jails_ data, we should rename the existing corrections-focused function. So we should rename `generateKeyInsightsData` to `generateCorrectionsKeyInsightsData`.

**Comment by nasaownsky** [comment]
> Welcome, @nasaownsky! Glad to have you on board.

Hey @jessex! Glad to be part of the team :)

File: src/utils/generateJailsKeyInsightsData.js
Comment (jessex) [inline src/utils/generateJailsKeyInsightsData.js:16]: Can we add unit tests for this file before we merge this PR?

File: src/utils/generateJailsKeyInsightsData.js
Comment (jessex) [inline src/utils/generateJailsKeyInsightsData.js:27]: What is this docstring attached to? Is it supposed to be attached to `generateJailsKeyInsightsData` below?

**Review by jessex — CHANGES_REQUESTED** [review]
Excited to work with you, @nasaownsky. I left a few comments and questions. From local testing, I have two brief remarks:

1. Can we remove the "Law Enforcement" tab for now? I know it's in the mocks, and I know it predates this PR, but we won't be enabling support for that for a while, so we should go ahead and remove the tab.
2. It looks like the "Jails" tab doesn't activate when you click on it in a mobile UI. If I open up Chrome's developer tools and choose a phone's layout, e.g. Pixel 2 XL, and click on "Jails", nothing happens.

File: src/utils/generateJailsKeyInsightsData.js
Comment (nasaownsky) [reply src/utils/generateJailsKeyInsightsData.js:27]: Yes, it is for `generateJailsKeyInsightsData`. Moved it below.

File: craco.config.js
Comment (nasaownsky) [reply craco.config.js:7]: No, I just didn't know that we can't use `.jsonl` for now. Switched to previous method.

File: src/App.js
Comment (nasaownsky) [reply src/App.js:89]: Fixed

File: src/utils/generateJailsKeyInsightsData.js
Comment (nasaownsky) [reply src/utils/generateJailsKeyInsightsData.js:16]: Added

**Review by jessex — APPROVED** [review]
Looks great to me, @nasaownsky! Congrats on your first PR.

Will merge when @Wmaileq approves.

File: src/utils/toFloat.js
Comment (Wmaileq) [inline src/utils/toFloat.js:17]: Why do not use just standard `parseFloat` function?

File: src/data/index.js
Comment (Wmaileq) [inline src/data/index.js:1]: Add copyright block here, please

File: src/components/MainPage/constants.js
Comment (Wmaileq) [inline src/components/MainPage/constants.js:1]: Add copyright block here, please.

File: src/components/MainPage/Tabs.js
Comment (Wmaileq) [inline src/components/MainPage/Tabs.js:5]: Add copyright block here, please.

**Review by Wmaileq — APPROVED** [review]
@nasaownsky Looks good to go now!
Let's fix minor problems mentioned in the comments and merge without waiting for me.

**Pull request merged**
